### ISSUE TITLE
Pillow - Generate a certificate template

### ISSUE THREAD

**Issue by jravenel** [jupyter-naas/awesome-notebooks#1212]

**Comment by SuhasBRao** [comment]
Hey @jravenel , I would like to work on this. I'm assigning the issue myself.

**Comment by SuhasBRao** [comment]
Yes we need a template png , I thought of creating a template myself using canva.
If you have any templates do share it, I'll use the same.

**Comment by jravenel** [comment]
Perfect if you want to do it with canva ✌️just make sure the png is accessible via internet by uploading the file in this ISSUE you can reuse the link. Good for you?

**Comment by SuhasBRao** [comment]
@jravenel can you please check out these two templates, I have [here](https://www.canva.com/design/DAFODmTkIdM/dduF9Gzgf3XHmh8sBv3NKw/edit?utm_content=DAFODmTkIdM&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton).
and let me know which one shall i go with?

**Comment by jravenel** [comment]
@SuhasBRao I checked but I think there was some stuff missing so I did it a simulation quickly on my side taking your contribution for yesterday as an example, here is the result: 

![Image](https://user-images.githubusercontent.com/21052349/193794492-723c3ee1-3034-4b8f-a2aa-bf7835d50be1.png)

**Comment by jravenel** [comment]
If it's ok for you, here is the boilerplate on which you should add the text and formatting: 


![Image](https://user-images.githubusercontent.com/21052349/193795091-42614e9c-825c-4d60-8913-c9af262e70e0.png)

**Comment by SuhasBRao** [comment]
It looks great, I will use this template itself.

**Comment by jravenel** [comment]
Cool 👍
